src/index.ts

```diff
@@ -14,22 +17,21 @@ function aleatorioBoolean(): boolean {
   }
 }
 
-function cargarVector(v: boolean[], cantidad: number) {
-  let indice: number;
-  for (indice = 0; indice < cantidad; indice++) {
-    v[indice] = aleatorioBoolean();
+function cargarVector(vector: boolean[], cantidad: number) {
+  for (let indice = 0; indice < cantidad; indice++) {
+    vector[indice] = aleatorioBoolean();
   }
 }
```

Comment:
- Cambiamos la **v** por **vector** para que sea mas representativo el nombre de la variable.
- El `let indice: number;` lo sacamos y directamente ponemos `let indice = 0` dentro del for.